.github/workflows/test.yml

```diff
@@ -33,7 +33,7 @@ jobs:
         uses: <action to setup tooling>
```

Comment:
This probably needs filling in

Reply:
There is no predefined sqlite action I could find. I better delete it.